gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Cluster.java

```diff
@@ -244,9 +241,15 @@ public static Cluster open(final Configuration conf) {
      * Create a {@code Cluster} using a YAML-based configuration file.
```

Comment:
Could you please expand the javadoc here to explain that the method will first try to read the file from the file system and then from resources?

Reply:
thanks for the reply! i will do the modification later for the points you given

Reply:
Could you please check the commit again. Wether it is ok to approve?